Pyroscope/Pyroscope.OpenTelemetry/PyroscopeSpanProcessor.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System.Diagnostics;
+using OpenTelemetry;
+
+namespace Pyroscope.OpenTelemetry;
+
+public class PyroscopeSpanProcessor : BaseProcessor<Activity>
```

Comment:
The history got last due to the renaming, but the changes here are
- we no longer read a config, which simplifies construction
- we don't restore the previous (parent) "profile_id" anymore in `OnEnd()`